tools/idf_monitor.py

```diff
@@ -220,6 +245,20 @@ class Monitor(object):
     """
     def __init__(self, serial_instance, elf_file, make="make", toolchain_prefix=DEFAULT_TOOLCHAIN_PREFIX, eol="CRLF"):
         super(Monitor, self).__init__()
+        posix_is_wsl = False
```

Comment:
What do you think about naming this running_on_wsl to match the field used in the Console instance? Another change that makes it easier to search for all of the WSL-related hacks at once. :)